### PR TITLE
New package: vera++

### PR DESCRIPTION
**Pull request by ofry**
New package: vera++
Maybe it will solve #5804

Please review.

### CODE REVIEW

File: mingw-w64-vera++/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Maintainer: David Demelier <markand@malikania.fr
+_realname=vera++
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=1.3.0
+pkgrel=2
```
Comment (oscarfv) [inline mingw-w64-vera++/PKGBUILD]: pkgrel starts from 1 and increases with every modification that requires a rebuild within a given `pkgver`.

Comment (ofry) [reply mingw-w64-vera++/PKGBUILD]: @oscarfv Done.

**Comment by ofry** [comment]
@oscarfv Could it be merged now?

**Comment by oscarfv** [comment]
You added a new commit on top of the previous one, while I suggested that you replace the previous commit with a new one with the extra modifications applied.

Since this project does not enforce strict commit practices, I guess your contribution will be merged anyways. You need to wait for the project's maintainer to approve it, though.

**Pull request merged**